src/pages/Ingame/index.js

```diff
@@ -151,7 +151,7 @@ export default function Game() {
       {isRuleModal && <RuleBook handleClick={handleToggleRule} />}
       {isBoardModal && selectBoard ? (
         <Modal
-          handleClick={handleCurrentBoard}
+          handleToggleBoard={handleCurrentBoard}
           isInterim={isCurrentModal}
           isfinishResult={isfinishResult}
```

Comment:
진행 중이 아니고 아예 끝났을 경우를 판단하는 변수명인 Bool이라면 아래와 같이 해주면 좋을 것 같습니다. 
아래 참고자료 링크보고 참조하시면 도움이 될거 같아서 남겨둡니다. 
참고자료: https://soojin.ro/blog/naming-boolean-variables
```suggestion
          isfinishedResult={isfinishedResult}
```

Reply:
넵 참고해보겠습니다

Reply:
*isFinishedResult로 변경했던 것 같은데 혹시 충돌있으려나요..